File: ClassChatIOS/ClassChat/ClassChat/Question.swift
//
//  Question.swift
//  ClassChat
//
//  Created by Jake on 3/18/15.
//  Copyright (c) 2015 Jake. All rights reserved.
//

import Foundation

class Question {
    
    
    var content: String!
    var commentList = [Comment]()
    var likes: Int!
    var id: Int!
    var user_id: Int!
    var last_updated_at: String!
    
    init(content: String, id: Int, user_id: Int, likes: Int, last_updated_at: String) {
        self.content = content
        self.likes = likes
        self.id = id
        self.user_id = user_id
        self.last_updated_at = last_updated_at
    }
    
}

File: ClassChatAndroid/app/src/main/java/com/example/CC/Question.java
package com.example.CC;

/**
 * Created by Jack on 4/12/15.
 */
public class Question {

    String content;
    int id;
    String user_name;
    int created_at;
    int user_id;
    int course_id;
    int likes;

    public Question(String content, int id, String user_name, int created_at, int user_id, int course_id,int likes){
        this.content = content;
        this.id = id;
        this.user_name = user_name;
        this.created_at = created_at;
        this.user_id = user_id;
        this.course_id = course_id;
        this.likes = likes;

    }
}
